src/content/getusermedia/record/js/main.js

```diff
@@ -99,37 +80,35 @@ function handleDataAvailable(event) {
   }
 }
 
-function handleStop(event) {
-  console.log('Recorder stopped: ', event);
-}
-
 function startRecording() {
   recordedBlobs = [];
   let options = {mimeType: 'video/webm;codecs=vp9'};
   if (!MediaRecorder.isTypeSupported(options.mimeType)) {
-    console.log(options.mimeType + ' is not Supported');
+    console.log(options.mimeType, ' is not Supported');
```

Comment:
running https://github.com/cpojer/js-codemod/blob/master/transforms/template-literals.js on the whole codebase might be easier :-)

Reply:
Yeah, I use both since I sometimes want the object displayed and sometimes just a string. This should probably be a template literal..